cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -30,15 +30,18 @@ func (w *wizard) promptTranscoderConfig() (float64, float64, *big.Int) {
 	fmt.Printf("Enter price per segment (default: 1) - ")
 	pricePerSegment = w.readDefaultBigInt(big.NewInt(1))
 
-	return blockRewardCut, feeShare, pricePerSegment
+	fmt.Printf("Enter service URI - ")
+	serviceURI := w.readString()
```

Comment:
Should we validate the URI here to make sure it's actually usable?  Otherwise a typo here can be expensive.

Reply:
Setting the Service URI is probably going to require a good bit of education and documentation for new and existing transcoders, so they understand what this entails. We could auto-select the value [0] and have the user confirm, to make it easier all-around. Wasn't expecting that feature in this basic PR though.

About HTTPS, it's actually easier to get a combined grpc+plain-http handler on the same port if TLS is enabled. We already generate self-signed certs at runtime [1], and set the client to ignore them [2]. Once we settle on an approach for https://github.com/livepeer/go-livepeer/issues/437 then we can start checking the certs rather than ignoring them.

[0] By finding the transcoder's own public address, then prefixing the default scheme. Also, print a loud warning if the transcoder appears to be NAT'd.
[1] https://github.com/livepeer/go-livepeer/pull/456/commits/5627be7d537cc46bf3016f01789cc374d9debb26
[2] https://github.com/livepeer/go-livepeer/pull/447/commits/066dcf7e081657958c0966f1245bcfe61c222689#diff-44bb03d888e11d6429223a7b0e55220dR204

Reply:
Agreed that there should definitely be validation for the service URI since the contract will store arbitrary strings. For starters, we could probably use [url.ParseRequestURI](https://golang.org/pkg/net/url/#ParseRequestURI) just to check if the URI is valid